test/curl.sh

```diff
@@ -47,3 +62,6 @@ check 127.0.0.1:5050 405
 # sync server
 # address of the endpoint have to be the same as a public_url in settings
 check localhost:5000
+
+# redis server
+check_redis
```

Comment:
@vladikoff added a test to make sure redis is up
Reference:
Using ping : https://stackoverflow.com/questions/21708107/how-do-i-check-if-redis-is-running-before-i-start-flask-and-start-it-if-it-isn

nc for pinging redis : https://stackoverflow.com/a/31195882